mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.tmpl

```diff
@@ -184,7 +184,6 @@ func TestAccContainerCluster_withAddons(t *testing.T) {
 				ResourceName:            "google_container_cluster.primary",
 				ImportState:             true,
 				ImportStateVerify:       true,
-				// TODO: clean up this list in `4.0.0`, remove both `workload_identity_config` fields (same for below)
```

Comment:
I tested without the `workload_identity_config {}` blocks, and got an error that they were required for a couple of the addons that are enabled.